File: common-core/src/main/java/com/ityu/service/cms/ContactsService.java
package com.ityu.service.cms;



import com.ityu.bean.entity.cms.Contacts;
import com.ityu.dao.cms.ContactsRepository;
import com.ityu.service.base.BaseService;
import org.springframework.stereotype.Service;

@Service
public class ContactsService extends BaseService<Contacts,Long, ContactsRepository> {
}

File: common-core/src/main/java/com/ityu/common/service/cms/ContactsService.kt
package com.ityu.common.service.cms

import com.ityu.common.bean.entity.cms.Contacts
import com.ityu.common.dao.cms.ContactsRepository
import com.ityu.common.service.BaseService
import org.springframework.stereotype.Service

@Service
open class ContactsService : BaseService<Contacts?, Long?, ContactsRepository?>()
